docs/troubleshooting.rst

```diff
@@ -1,34 +1,55 @@
 Troubleshooting
 ===============
 
-Using supervisord for development
----------------------------------
+My log parsing isn't working
+----------------------------
 
-On an ubuntu machine you can install supervisord with
+If you encounter job log parsing not working in general, or when selecting a job, you should ``ctrl-c`` to do a warm shutdown of your celery process then:
 
 .. code-block:: bash
 
-   >sudo apt-get install supervisor
+    (venv)vagrant@local:~/treeherder$ rm celerybeat-schedule
 
-To start supervisord with an arbitrary configuration, you can type:
+And then restart your celery worker as normal via:
 
 .. code-block:: bash
 
-   >supervisord -c my_config_file.conf
+    (venv)vagrant@local:~/treeherder$ celery -A treeherder worker -B
 
-You can find a supervisord config file inside the deployment/supervisord folder.
-That config file contains a section for each service that you may want to run.
-Feel free to comment one or more of those sections if you don't need that specific service.
-If you just want to access the restful api or the admin for example, comment all those sections but the one
-related to gunicorn.
-You can stop supervisord (and all processes he's taking care of) with ctrl+c.
-Please note that for some reason you may need to manually kill the celery worker when it's under heavy load.
+Can I use supervisord instead for my services?
+----------------------------------------------
+
+Yes, you can install supervisord in your environment with:
+
+.. code-block:: bash
+
+   (venv)vagrant@local:~/treeherder$ pip install supervisor
```

Comment:
Could we not install this by default in the vagrant env and save our users some typing?